### PR TITLE
sop-039-nl

### PR DESCRIPTION
**Pull request by wlangera**
## Description

Deze procedure beschrijft het stappenplan om MAS data te downloaden via de WFS-server van Sovon en deze data te exporteren voor verdere analyse. Dit wordt uitgevoerd via een specifieke workflow in QGIS.

In collaboration with @hansvancalster 

## Task list

Steps by contributor:

-   [x] Add description to this pull request (under "## Description")
-   [x] Open the dropdown (triangle) near 'create pull request' and choose 'Create draft pull request'
-   [x] Check for potential problems by running `protocolhelper::check_frontmatter()` and address them
-   [x] Check for potential problems by running `protocolhelper::check_structure()` and address them
-   [x] Add further commits if needed and push them to GitHub
-   [x] Update the protocol-specific `NEWS.Rmd`
-   [x] Mark the pull request as 'ready for review'

Review steps for the author(s):

-   [x] Add reviewers, at least one subject-matter specialist and one administrator
-   [x] Wait for review comments and address them
-   [x] Iterate until reviewer approvals (merging the pull request will be done by an administrator)
-   [x] Verify that the checks done by continuous integration succeeded. These will check if `protocolhelper::check_frontmatter()` and `protocolhelper::check_structure()` succeeded without errors.

To be done by an administrator after review: see [guidelines for admins](https://github.com/inbo/protocolsource/blob/main/RELEASES.md).

### CODE REVIEW

**Comment by wlangera** [comment]
@hansvancalster can you check this protocol before I add reviewers?
I get one error :

```r
protocolhelper::check_structure("sop-039-nl")
#> Errors in protocol sop-039-nl:
#> - Heading(s) ## Subtitel 1 lack(s) in file 06_werkwijze.Rmd
#> - Heading(s) ## Subtitel 2 lack(s) in file 06_werkwijze.Rmd
```

I have subtitles in that file. So I am not sure what the problem is.

**Comment by hansvancalster** [comment]
> ```r
> protocolhelper::check_structure("sop-039-nl")
> #> Errors in protocol sop-039-nl:
> #> - Heading(s) ## Subtitel 1 lack(s) in file 06_werkwijze.Rmd
> #> - Heading(s) ## Subtitel 2 lack(s) in file 06_werkwijze.Rmd
> ```
> 
> I have subtitles in that file. So I am not sure what the problem is.

This is a "bug" in protocolhelper. It's because these placeholders are part of the Rmarkdown [templates](https://github.com/inbo/protocolhelper/blob/main/inst/rmarkdown/templates/template_sop_nl/skeleton/06_werkwijze.Rmd) and check_structure checks if all template headers are present in protocol files... So I should just remove the placeholders from the template (or move them inside the commented out section). Just ignore this for now. I will need to update the protocolhelper package to fix this.

File: source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+**MAS:** Het MAS, of Meetnet Agrarische Soorten, monitort broedvogels (en zoogdieren) in agrarische gebieden om trends in aantallen en verspreiding vast te stellen, en de effectiviteit van beschermingsmaatregelen te evalueren [@MAS].
+Dit meetnet wordt reeds toegepast in Nederland en sinds de opstart van het MBAG project ook in Vlaanderen.
+
+**MBAG:** Het MBAG project, of Meetnet Biodiversiteit in het Agrarisch Gebied, is een initiatief van het Instituut voor Natuur- en Bosonderzoek (INBO) in Vlaanderen dat gericht is op het monitoren en analyseren van de biodiversiteit in landbouwgebieden.
+Het project richt zich op het verkrijgen van inzicht in de biodiversiteit, met een focus op akkervogels, bestuivers, bodembiodiversiteit, en landschapselementen, om zo bij te dragen aan zowel natuurbehoud als duurzame landbouwpraktijken [@MBAG].
+
+**QGIS:** QGIS is een open-source geografisch informatiesysteem waarin je geografische gegevens importeren, bekijken, bewerken, analyseren en visualiseren [@QGISSoftware].
```
Comment (EmmaCartuyvels1) [inline source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd:18]: ...waarin je geografische gegevens **kan** importeren...

**Review by EmmaCartuyvels1 — COMMENTED** [review]
Very clear documentation @wlangera!

The first two links don't work when using the file locally but I think that's normal?

File: source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd (hunk 1 above)
Comment (wlangera) [inline source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd:18]:
```suggestion
**QGIS:** QGIS is een open-source geografisch informatiesysteem waarin je geografische gegevens kan importeren, bekijken, bewerken, analyseren en visualiseren [@QGISSoftware].
```

Comment (wlangera) [reply source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd]: What do you mean with "the first two links"?

Comment (wlangera) [reply source/sop/sop_039_nl_download_mas_wfs/03_definities_afkortingen.Rmd]: @EmmaCartuyvels1 kan je hier nog op terugkomen of approven?

**Review by EmmaCartuyvels1 — APPROVED** [review]
All clear!

**Comment by wlangera** [comment]
@hansvancalster I think we are set. Can you publish this or let me know if I need to do anything else?

**Review by hansvancalster — APPROVED** [review]

**Pull request merged**